vercel.json

```diff
@@ -0,0 +1,9 @@
+{
+  "buildCommand": "./gradlew :docs-website:build",
+  "github": {
+    "silent": true,
+    "autoJobCancelation": true
+  },
+  "installCommand": "amazon-linux-extras install python3.8 && py3=\"$(which python3)\" && rm \"$py3\" && ln \"$(which python3.8)\" \"$py3\" && ./metadata-ingestion/scripts/install_deps.sh && yum install -y gcc python38-devel",
```

Comment:
How come we're using python3.8 here? And what is running this command?

Reply:
this command is being run by vercel, which is a third-party tool